ui/src/react-admin/modules/content-page/services/content-page.service.ts

```diff
@@ -727,7 +728,19 @@ export class ContentPageService {
 		let url: string | undefined;
 		let body: any | undefined;
 		try {
-			url = `${AdminConfigManager.getConfig().database.proxyUrl}/admin/content-pages`;
+			switch (AdminConfigManager.getConfig().database.databaseApplicationType) {
+				case AvoOrHetArchief.hetArchief:
+					url = `${AdminConfigManager.getConfig().database.proxyUrl}/admin/content-pages`;
+					break;
+
+				case AvoOrHetArchief.avo:
+					url = `${AdminConfigManager.getConfig().database.proxyUrl}/content-pages`;
+					break;
+
+				default:
+					throw new Error('Could not determine URL to poll for content-pages');
+			}
```

Comment:
Shouldn't it be the clients responsibility to set the correct url in database.proxyUrl?
so we don't have to do this check here?

Maybe we should rename the config value to?
```
AdminConfigManager.getConfig().adminCoreApiUrl
```

There shouldn't be any code inside the admin-core-ui that calls routes on the regular proxy url. Admin-core-ui should only call routes on the admin-core-api endpoint

But maybe its better to change this after merging this PR, to avoid more merge conflicts?